lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -24,7 +24,7 @@ def process_vm_opts(opts)
           end
           # rubocop:enable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
 
-          # rubocop:disable Metrics/AbcSize
+          # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
With this simpler code, is it still needed to disable these cops?

Reply:
yes, rubocop still wants to disable it.